gradle/javadoc.gradle

```diff
@@ -54,9 +54,7 @@ task mockitoJavadoc(type: Javadoc) {
     """.replaceAll(/\r|\n/, ""))
     options.stylesheetFile file("src/javadoc/stylesheet.css")
 //    options.addStringOption('top', 'some html')
-    if (JavaVersion.current().isJava8Compatible()) {
-        options.addStringOption('Xdoclint:none', '-quiet')
-    }
+    options.addStringOption('Xdoclint:none', '-quiet')
```

Comment:
Yup in https://github.com/mockito/mockito/commit/be755a3d9e8424e8587857390abbc07001644778 :smile: